src/bundle/Resources/views/themes/admin/ui/component/summary-tile/summary-tile.html.twig

```diff
@@ -0,0 +1,23 @@
+{% import "@ibexadesign/ui/component/macros.html.twig" as html %}
+
+{% set attr = attr|default({})|merge({
+    class: ('ibexa-summary-tile '
+        ~ (color ? 'ibexa-summary-tile--' ~ color ~ ' ')
```

Comment:
maybe sth like `icon_color`/`circle_color` would be better? as this variable is only responsible for this, and `color` imho indicates sth more